src/crimson/os/seastore/segment_cleaner.h

```diff
@@ -260,14 +307,18 @@ class SpaceTrackerSimple : public SpaceTrackerI {
     segment_id_t segment,
     seastore_off_t offset,
     extent_len_t len) final {
-    return update_usage(segment, len);
+    return update_usage(
+        segment,
+        len + ceph::encoded_sizeof_bounded<extent_info_t>());
```

Comment:
This doesn't make sense to me -- it seems like the caller would already have needed to include extent_info_t?  This also seems to break a symmetry that update_usage is always called with aligned values.

Reply:
My thinking was that every extent needs an `extent_info_t` in the current implementation, which will not be discarded after journal trimming, and will still be needed during reclaim/rewrite. So I think it might be more accurate to account this metadata usage in `stats.used_bytes` and the segment usage from `SpaceTracker`. What do you think?

Reply:
>  it seems like the caller would already have needed to include extent_info_t.

Yeah, I think it is reasonable to move this up to the callers as part of the metadata usage.